pkg/fleet-manager/fleet_plugin.go

```diff
@@ -38,7 +38,7 @@ const (
 
 	NoneClusterIP = "None"
 
-	FleetPluinLabel = "fleet.kurator.dev/fleet-plugin"
+	FleetPluinLabel = "fleet.kurator.dev/plugin-typo"
```

Comment:
`typo`

Reply:
I found this name already been defined
see https://github.com/kurator-dev/kurator/blob/5734bb72d2d0a2b8fc00d974e41da5f17560240a/pkg/fleet-manager/util.go#L31C1-L31C1
so I desire to remove it